RunningOrder/Sprint/Views/SprintList.swift

```diff
@@ -18,32 +18,56 @@ extension SprintList {
         @ObservedObject var logic: Logic
         let space: Space
 
+        private func setupSprintDisplay(_ sprint: Sprint) -> some View {
+            return NavigationLink(
+                destination: StoryList(sprint: sprint),
+                label: {
+                    HStack {
+                        SprintNumber(
+                            number: sprint.number,
+                            colorIdentifier: sprint.colorIdentifier
+                        )
+                        Text(sprint.name)
+                    }
+                }
+            )
+        }
+
+        private func setupDeleteButton(_ sprint: Sprint) -> some View {
```

Comment:
please rename the method : 
```suggestion
        private func deleteButton(_ sprint: Sprint) -> some View {
```

Reply:
instead of these two methods, what do you think of creating a 
```swift
struct SprintRow: View {
    let sprint: Sprint

    var body: some View {
        NavigationLink(
            destination: StoryList(sprint: sprint),
            label: {
                HStack {
                    SprintNumber(
                        number: sprint.number,
                        colorIdentifier: sprint.colorIdentifier
                    )
                    Text(sprint.name)
                }
            })
            .contextMenu {
                if !sprint.closed {
                    Button(
                        action: { logic.closeSprint(sprint) },
                        label: { Text("Close Sprint") }
                    )
                }

                Button(
                    action: { logic.deleteSprint(sprint) },
                    label: { 
                        Text("Delete Sprint")
                            .foregroundColor(.red)
                    }
                )
    }
```

Reply:
this way everything is mutualized in a row type, and the button is just shown when necessary